### DISCUSSION TITLE
Custom BOM Excel Template

### DISCUSSION THREAD

**Discussion by JacobEFO** [inventree/InvenTree#8316] [Q&A]
Hi InvenTree users and developers,

I was wondering if there is any way you can create a custom template for Excel BOM exports. I have not been able to find any such functionality, although a BOM report HTLM template can be customized through the Admin Center.

Do you have any pointers to this particular use-case?

**Comment by SchrodingersGat** [comment]
@JacobEFO I have been wondering if there would be a good opportunity to extend the current "reporting" structure to generate other sorts of files - spreadsheet reports being a good example. Perhaps that would be the way to go about this?

**Reply by JacobEFO** [reply]
I have not yet figured out how to exactly utilize the Reporting tool, but I imagine it could be quite nice.

At least for me personally, I sometimes like to be able to view the BOM in an Excel format (which is already supported outside the reporting tool), so have options for the report would be a nice-to-have feature. And changing the outcome also looks fairly simple.
